src/app/view/components/dot-dialog/dot-dialog.component.scss

```diff
@@ -4,11 +4,12 @@
 .dialog ::ng-deep {
     .ui-dialog {
         max-width: 1200px;
+        top: auto !important;
         width: 95vw;
 
         .ui-dialog-content {
             overflow: hidden;
-            height: 80vh !important;
+            height: 70vh !important;
```

Comment:
I don't think this is a good approach... We are going to need to find a way to set the size of the dialog from where we use it, because not all dialog across the app will be the same size.